cluster/manifests/deployment-service/01-config.yaml

```diff
@@ -15,6 +15,7 @@ data:
   oidc-subject-key: "{{.Cluster.LocalID}}.{{.Cluster.Alias}}.zalan.do:sub"
   s3-bucket-name: "zalando-deployment-service-{{accountID .Cluster.InfrastructureAccount}}-{{.Cluster.LocalID}}"
   status-service-url: "https://deployment-status-svc-{{.Cluster.Alias}}.{{.Values.hosted_zone}}"
+  status-service-url-local: "http://deployment-status-service.ingress.cluster.local."
```

Comment:
This is just a small optimization to tell the DNS resolver to not add search domains as this is the full name (this could ofc. be added in a lot of places).

Reply:
I see. Thanks 🙏